src/connection.ts

```diff
@@ -3094,10 +3120,7 @@ class Connection extends EventEmitter {
     }
   }
 
-  /**
-   * @private
-   */
-  makeRequest(request: Request | BulkLoad, packetType: number, payload: (Iterable<Buffer> | AsyncIterable<Buffer>) & { toString: (indent?: string) => string }) {
+  private makeRequest(request: Request | BulkLoad, packetType: number, payload: (Iterable<Buffer> | AsyncIterable<Buffer>) & { toString: (indent?: string) => string }) {
```

Comment:
Do you mind switching this back to not be `private`? This is breaking some unrelated code.

Reply:
interesting, it fails a compile if I keep it as a JS Doc though, specifically 'type annotaiton not expected'